File: 01nicollypatricia/Códigos em javascript/primo.js
function numeroPrimo(numero) {
    let primo = true;
    for(let contador = 2; contador < numero; contador++) {
      if(numero % contador === 0) {
        primo = false;
        break;
      }
    }
    if(primo == true) {
     console.log(`O valor ${numero} é primo!`);
    } else {
      console.log(`O valor ${numero} não é primo!`);
    }
   }

let c = numeroPrimo(4);

File: 01nicollypatricia/Java/src/Primo.java
import java.util.Scanner;
public class Primo {

		public void primo() {
			Scanner scan = new Scanner(System.in);
			int num;
			int numDivisores=0;
			
			System.out.print("Informe um número: ");
			num = scan.nextInt();
			
			for(int i=1; i<=num; i++) {
				if(num % i == 0) {
					numDivisores++;
					}
				}
			if(numDivisores == 2) {
				System.out.print("O numero eh primo");
				} else {
				System.out.print("O numero não eh primo");
				}
							scan.close();
		}
		}

